app/config/FrontendAppConfig.scala

```diff
@@ -130,4 +134,11 @@ class FrontendAppConfig @Inject()(environment: Environment, implicit val runMode
 
   // Manage VAT Subscription Frontend
   override lazy val manageVatUrl: String = getString(ConfigKeys.manageVatHost) + getString(ConfigKeys.manageVatUrl)
+
+  override def languageMap: Map[String, Lang] = Map(
+    "english" -> Lang("en"),
+    "cymraeg" -> Lang("cy")
+  )
+  //TODO the controllers to switch the language
+  override val routeToSwitchLanguage: String => Call = (lang: String) => controllers.routes.HelloWorldController.helloWorld()
```

Comment:
#Language Controller
GET         /language/:lang                                  controllers.LanguageController.switchToLanguage(lang: String)
will need to be added to app.routes

And routeToSwitchLanguage will need to be changed to 
override val routeToSwitchLanguage: String => Call = (lang: String) => controllers.routes.LanguageController.switchToLanguage(lang)

Reply:
Done